integrationtests/as-integration-client/build.xml

```diff
@@ -51,44 +51,128 @@
       <echo message="Infinispan server started"/>
    </target>
 
-   <target name="kill_server" depends="checkSkip" unless="testsSkipped">
-      <exec executable="${server.jvm}/bin/jps" output="jps.pid" osfamily="unix"/>
-      <loadfile srcfile="jps.pid" property="pid" failonerror="false">
-         <filterchain>
-            <linecontains>
-               <contains value="jboss-modules.jar"/>
-            </linecontains>
-            <tokenfilter>
-               <deletecharacters chars="jboss-modules.jar"/>
-               <ignoreblank/>
-            </tokenfilter>
-            <striplinebreaks/>
-         </filterchain>
-      </loadfile>
-      <exec executable="netstat" output="jps.pid" osfamily="windows">
-         <arg line="-aon"/>
-      </exec>
-      <loadfile srcfile="jps.pid" property="pid" failonerror="false">
-         <filterchain>
-            <linecontains>
-               <contains value="LISTENING"/>
-               <contains value=":11222"/>
-            </linecontains>
-            <tokenfilter>
-               <replaceregex pattern=".*LISTENING([ \t]+)([0-9]+)" replace="\2"
-                             flags="gi"/>
-               <ignoreblank/>
-            </tokenfilter>
-            <striplinebreaks/>
-         </filterchain>
-      </loadfile>
-      <echo message="Killing Infinispan server with PID - ${pid}"/>
-      <exec executable="kill" osfamily="unix">
-         <arg line="-9 ${pid}"/>
-      </exec>
-      <exec executable="taskkill" osfamily="windows">
-         <arg line="/F /T /PID ${pid}"/>
-      </exec>
-      <delete file="jps.pid"/>
-   </target>
+    <target name="-do.kinit" depends="-check.skipped" unless="tests.skipped">
+        <exec executable="bash" output="ps.pid" resultproperty="ps.result" failifexecutionfails="false" failonerror="false" osfamily="unix">
+            <arg value="-c"/>
+            <arg value="ps -eaf --columns 20000 | grep jboss-modules.jar | grep -v -w grep | awk '{print $$2;}'"/>
```

Comment:
`ps -fCjava` should filter by command name and avoid the need for `grep -v -w grep` - unless `-C` is not supported on one of the OSes we test on.